allensdk/brain_observatory/behavior/behavior_session.py

```diff
@@ -432,7 +494,8 @@ def stimulus_templates(self, value):
 
     @property
     def stimulus_timestamps(self) -> np.ndarray:
-        """Get stimulus timestamps from pkl file.
+        """Timestamps associated with the stimulus presetntation on
+        the monitor retrieved from pkl file. Sampled at 60Hz.
```

Comment:
I think we should try to avoid referring to the 'pkl file'. That's not meaningful to external users. Instead, I'd suggest we simply call it the 'stimulus file'. Or, maybe to be more specific, the 'data file saved at the end of the behavior session'.

Reply:
I've updated the docstrings so they no longer reference the pkl files they now say "data file saved at the end of the behavior session"